pytest_fixtures/component/activationkey.py

```diff
@@ -6,8 +6,14 @@
 
 
 @pytest.fixture(scope='module')
-def module_activation_key(module_org, module_target_sat):
-    return module_target_sat.api.ActivationKey(organization=module_org).create()
+def module_activation_key(module_entitlement_manifest_org, module_target_sat):
+    activation_key = module_target_sat.api.ActivationKey(
+        auto_attach=True,
```

Comment:
`auto_attach` shouldn't be needed in case of SCA

Reply:
Removed.